fastlane/Fastfile

```diff
@@ -269,41 +269,67 @@ platform :ios do
   end
 
   #####################################################################################
-  # finalize_release
+  # check_all_translations
   # -----------------------------------------------------------------------------------
-  # This lane finalize a release: updates store metadata, bump final version number,
-  # remove branch protection and close milestone, then trigger the final release on CI
+  # This lane checks the translation progress for all the projects in GlotPress.
+  #
+  # By default, it will print all values. You can make it ask whether to continue when
+  # if found translations that are below the threshold by passing `interactive: true`.
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane finalize_release [skip_confirm:<skip confirm>] [version:<version>]
+  # bundle exec fastlane check_all_translations [interactive:<interactive>]
   #
   # Example:
-  # bundle exec fastlane finalize_release
-  # bundle exec fastlane finalize_release skip_confirm:true
+  # bundle exec fastlane check_all_translations
+  # bundle exec fastlane check_all_translations interactive:true
   #####################################################################################
-  desc 'Trigger the final release build on CI'
-  lane :finalize_release do |options|
-    UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
+  desc 'Check translation progress for all GlotPress projects'
+  lane :check_all_translations do |options|
+    abort_on_violations = false
```

Comment:
Any reason to create a variable with that hardcoded value as opposed to just using `false` inline at call site on line 294?

(reminds me of [this tweet](https://twitter.com/nicklockwood/status/1421047660006494209?s=20))

Reply:
Good point. I was considering passing that parameter too, so I put it in a var in the meantime. Then, I decided not to, but left it there.

Nick's got a point in favoring inline literals. Although, I'd say in this case I like having the value extracted in a single place because, if we want to toggle it to `true`, we'll need to change only one line. I'm also aware that "if we want to..." is a smell for code that YAGNI. 💄 😅

Reply:
> I'd say in this case I like having the value extracted in a single place because, if we want to toggle it to `true`, we'll need to change only one line

I was reading through the Twitter thread, and saw [this tweet](https://twitter.com/nicklockwood/status/1421047662590234627?s=20) in the thread that says something similar:

> Constants serve two functions:
>
> 1) giving context to something that is otherwise unclear (e.g. a magic number)
>
> 2) **allowing a value to be re-used in multiple places without repetition**
>
> If a constant serves neither of those purposes, it's redundant. It makes the code worse.

Reply:
Yeah I only realized _after_ I wrote that comment that it was used in 3 call sites (the 3 translation checks) below, not just the one on line 294, and realized then why it could make sense to extract it after all, in order to toggle it more easily for all 3 at once, so all good 👍